controllers/directivebreakdown_controller.go

```diff
@@ -276,17 +276,6 @@ func (r *DirectiveBreakdownReconciler) Reconcile(ctx context.Context, req ctrl.R
 		if err != nil {
 			return ctrl.Result{}, err
 		}
-	case "container":
-		pinnedContainerProfile, err := createPinnedContainerProfile(ctx, r.Client, r.Scheme, argsMap, dbd, commonResourceName)
```

Comment:
Because a container profile doesn't need a directive breakdown. I can move it to the proposal phase if you like. It just doesn't make sense to me that proposal would _do_ stuff. But it's just a meaningless name at this point.

Reply:
Moved to Proposal stage